### PR TITLE
Update links for new issue tracker

### PR DESCRIPTION
**Pull request by hugovk**
Update old references to bugs.python.org to the new GitHub issue tracker.

Rather than linking directly to https://github.com/python/cpython/issues, I used https://devguide.python.org/tracker/ which has important info like to search for duplicates first.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ assignees: ''
 > Note: This repo is for the Python devguide. If you are requesting an
 enhancementfor the Python language or CPython interpreter,
 then the CPython issue tracker is better
-suited for this report: https://bugs.python.org
+suited for this report: https://devguide.python.org/tracker/
```
Comment (JelleZijlstra) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: I think it makes sense to link directly to https://github.com/python/cpython/issues here

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ assignees: ''
 > Note: This repo is for the Python devguide. If you are requesting an
 enhancement for the Python language or CPython interpreter,
 then the CPython issue tracker is better
-suited for this report: https://bugs.python.org
+suited for this report: https://devguide.python.org/tracker/
```
Comment (JelleZijlstra) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: And here

File: index.rst [hunk 3]
```diff
@@ -54,7 +54,7 @@ instructions please see the :ref:`setup guide <setup>`.
       git checkout -b fix-issue-12345 main
 
    If an issue does not already exist, please `create it
-   <https://bugs.python.org/>`_.  Trivial issues (e.g. typo fixes) do not
+   <https://devguide.python.org/tracker/>`_.  Trivial issues (e.g. typo fixes) do not
```
Comment (JelleZijlstra) [inline index.rst:57]: And here

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (hugovk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated x 3!

**Review by JelleZijlstra — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (JelleZijlstra) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks!

**Pull request merged**